pkg/injector/webhook.go

```diff
@@ -348,9 +356,6 @@ func getPartialMutatingWebhookConfiguration(cert certificate.Certificater, webho
 // It is necessary to perform this patch because the original MutatingWebhookConfig YAML does not contain the root certificate.
 func updateMutatingWebhookCABundle(cert certificate.Certificater, webhookName string, clientSet kubernetes.Interface) error {
 	mwc := clientSet.AdmissionregistrationV1beta1().MutatingWebhookConfigurations()
-	if err := webhookExists(mwc, webhookName); err != nil {
-		log.Error().Err(err).Msgf("Error getting MutatingWebhookConfiguration %s; Will not update CA Bundle for webhook", webhookName)
-	}
```

Comment:
we still would want to check for this condition ?

Reply:
I could keep it, I figured I'd remove as it could provide wrong assessment read by the wrong eyes. Idempotence ensured by code should take for granted things exist/don't, or not check at all. (but that's my opinion, ofc)

Reply:
I guess its okay to delete then